packages/experimental/Avatar/src/useAvatar.ts

```diff
@@ -11,14 +11,17 @@ import { getHashCodeWeb } from './getHashCode';
  * @returns configured props and state for FURN Avatar
  */
 export const useAvatar = (props: JSAvatarProps): AvatarInfo => {
-  const { avatarColor, active, accessibilityLabel, activeAppearance, badge, idForColor, initials, name, src, ring, shape, ...rest } = props;
+  const { active, accessibilityLabel, accessibilityHint, activeAppearance, badge, initials, name, src, ring, shape, ...rest } = props;
 
   const showRing = active === 'active' && activeAppearance === 'ring';
   const transparentRing = !!ring?.transparent;
   const showBadge = (!active || active === 'unset') && !!badge && !!badge.status;
+  const accessibilityText = `${name || ''}${showBadge ? `, ${badge.status}` : ''}`;
```

Comment:
I wonder if this is safe to hardcode? Delimiters and ordering of words/information can be different between languages, so hardcoding string building logic is generally advised against.